bin/near

```diff
@@ -12,6 +12,25 @@ const exitOnError = async function(promise) {
   }
 }
 
+//For smart contract:
```

Comment:
nit: let's have spaces after `//` 
probably should configure eslint to fix this